_includes/sections/vpn.html

```diff
@@ -85,6 +87,8 @@ <h5><span class="badge badge-danger">No Security Audit</span></h5>
     <p>IVPN has undergone a <a href="https://cure53.de/audit-report_ivpn.pdf">no-logging audit from Cure53</a> which concluded in agreement with IVPN's no-logging claim. However, IVPN has not undergone a more comprehensive security audit by an independent third party, and therefore cannot be strongly recommended at this time.</p><p>We have still chosen to list it on this page with the assumption that an audit will be published soon. <a href="https://nitter.net/ivpnnet/status/1181954975687163905">IVPN has hired Cure53</a> to undertake a comprehensive audit covering the IVPN website, public and internal server infrastucture. They expect the audit to begin in November 2019 and be completed by the 6 auditors in January 2020.</p>
     <h5><span class="badge badge-success">77+ Servers</span></h5>
     <p>IVPN has 77 servers in 31 countries at the time of writing this page. Typically the more servers a provider offers, the better. IVPN has a decent (but not exceptional) server count that will most likely provide adequate coverage to most users.</p>
+    <h5><span class="badge badge-success">Remote port forwarding</span></h5>
+    <p>Remote <a href="https://en.wikipedia.org/wiki/Port_forwarding">port forwarding</a> is possible with pro accounts. Port forwarding <a href="https://www.ivpn.net/knowledgebase/81/How-do-I-activate-port-forwarding.html">can be activated</a> via the client area.</p>
```

Comment:
Maybe we should mention that port forwarding is only available for OpenVPN and non-US servers (https://www.ivpn.net/knowledgebase/116/Port-forwarding-is-not-working-why.html)

Also, it seems that Pro is used as a proper noun (for example, they say it as "IVPN Pro plan" at https://www.ivpn.net/knowledgebase/20/How-many-devices-or-simultaneous-connections-can-I-have-to-IVPN.html)

```suggestion
    <p>Remote <a href="https://en.wikipedia.org/wiki/Port_forwarding">port forwarding</a> is possible with a Pro plan. Port forwarding <a href="https://www.ivpn.net/knowledgebase/81/How-do-I-activate-port-forwarding.html">can be activated</a> via the client area.</p>
```

Reply:
> Maybe we should mention that port forwarding is only available for OpenVPN and non-US servers (https://www.ivpn.net/knowledgebase/116/Port-forwarding-is-not-working-why.html)

The reason for that is they don't want people doing P2P on US servers, likely because of DMCA notices. I have noticed that with a few providers.

They are also doing Wireguard, so that might not be strictly true about it only being available with OpenVPN, in the future.

Reply:
Looking at https://mullvad.net/nl/help/port-forwarding-and-mullvad/ it seems that port forwarding over wireguard is certainly possible, so I think that's a fair point.